event-adapter/pkg/apiserver/installer/installer_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Please fix commit c82734ffbaf5c8473dfd92e80775064df90cd027 instead of re-introducing this file in another commit. This way I'm not able to easily see a diff for this file.